packages/theming/atlas/src/theme/web/exclusion-variables.scss

```diff
@@ -7,7 +7,7 @@ $exclude-button: false;
 $exclude-button-helpers: false;
 $exclude-check-box: false;
 $exclude-custom-dijit-widget: false;
-$exclude-custom-switch: false;
+$exclude-switch: false;
```

Comment:
Renaming this will break Mendix projects migrated to 9 already :/ Not sure how to approach this.

Reply:
Good catch. I decided to revert the name change. This is a good case in point regarding "syncing" themesource and theme, which is a larger problem, so addressing it elsewhere seems best.

Reply:
@Keraito if you can please review for Isa

Reply:
Addressed in https://github.com/mendix/widgets-resources/commit/cc8de45901a271d4bec9492d592d4662ed67044c